src/i18n/strings/en_EN.json

```diff
@@ -1426,9 +1426,15 @@
     "Send %(eventType)s events": "Send %(eventType)s events",
     "Permissions": "Permissions",
     "Select the roles required to change various parts of the room": "Select the roles required to change various parts of the room",
+    "Are you sure you want to add encryption to this public room?": "Are you sure you want to add encryption to this public room?",
+    "<b> It’s not recommended to add encryption to public rooms. </b>Anyone can find and join public rooms, so anyone can read messages in them. You’ll get none of the benefits of encryption, and you won’t be able to turn it off later. Encrypting messages in a public room will make receiving and sending messages slower.": "<b> It’s not recommended to add encryption to public rooms. </b>Anyone can find and join public rooms, so anyone can read messages in them. You’ll get none of the benefits of encryption, and you won’t be able to turn it off later. Encrypting messages in a public room will make receiving and sending messages slower.",
+    "To avoid these issues, create a <a> new encrypted room </a> for the conversation you plan to have.": "To avoid these issues, create a <a> new encrypted room </a> for the conversation you plan to have.",
     "Enable encryption?": "Enable encryption?",
     "Once enabled, encryption for a room cannot be disabled. Messages sent in an encrypted room cannot be seen by the server, only by the participants of the room. Enabling encryption may prevent many bots and bridges from working correctly. <a>Learn more about encryption.</a>": "Once enabled, encryption for a room cannot be disabled. Messages sent in an encrypted room cannot be seen by the server, only by the participants of the room. Enabling encryption may prevent many bots and bridges from working correctly. <a>Learn more about encryption.</a>",
     "This upgrade will allow members of selected spaces access to this room without an invite.": "This upgrade will allow members of selected spaces access to this room without an invite.",
+    "Are you sure you want to make this encrypted room public?": "Are you sure you want to make this encrypted room public?",
+    "<b> It’s not recommended to make encrypted rooms public. </b> It It will mean anyone can find and join the room, so anyone can read messages. You’ll get none of the benefits of encryption. Encrypting messages in a public room will make receiving and sending messages slower.": "<b> It’s not recommended to make encrypted rooms public. </b> It It will mean anyone can find and join the room, so anyone can read messages. You’ll get none of the benefits of encryption. Encrypting messages in a public room will make receiving and sending messages slower.",
```

Comment:
```suggestion
    "<b> It’s not recommended to make encrypted rooms public. </b> It will mean anyone can find and join the room, so anyone can read messages. You’ll get none of the benefits of encryption. Encrypting messages in a public room will make receiving and sending messages slower.": "<b> It’s not recommended to make encrypted rooms public. </b> It It will mean anyone can find and join the room, so anyone can read messages. You’ll get none of the benefits of encryption. Encrypting messages in a public room will make receiving and sending messages slower.",
```